cmd/trayscale/systray.go

```diff
@@ -2,26 +2,32 @@ package main
 
 import (
 	_ "embed"
+	"fmt"
 
+	"deedles.dev/trayscale/internal/tsutil"
 	"fyne.io/systray"
 )
 
 type tray struct {
-	showItem *systray.MenuItem
-	quitItem *systray.MenuItem
+	showItem     *systray.MenuItem
+	quitItem     *systray.MenuItem
+	selfNodeItem *systray.MenuItem
 }
 
 func initTray(online bool) *tray {
 	systray.SetIcon(statusIcon(online))
 	systray.SetTitle("Trayscale")
 
+	selfNodeItem := systray.AddMenuItem("", "")
+	systray.AddSeparator()
```

Comment:
It's disappointing that there doesn't seem to be a way to hide the separator, but at least the self item should be shown most of the time.

Reply:
Yup.

Since I've been playing around with listing other nodes, I can tell, that systray, dbus or Plasma does not work as expected.

Starting with submenu, if I don't add anything to the submenu, adding things later (on first `Update`) will not show any items in the submenu. If I add a placeholder, adding new items once will work, but adding new ones or trying to hide/disable them (e.g. when offline) does not work.

Hell, I can't even tell if current `Hide()` on `selfNodeItem` works.

My current conclusion: if things will be like this, it might make more sense to reset the menu on each change and redraw everything. With this we might be wasting some resources, but we will be in full control of entire structure at all times.